responsibleai/tests/databalanceanalysis/conftest.py

```diff
@@ -0,0 +1,190 @@
+            "PointwiseMutualInfo": {"0": 0.4906283898, "1": 1.0465727536},
+            "SorensonDiceCoeff": {"0": 0.1394139444, "1": 0.1622153797},
+            "JaccardIndex": {"0": 0.1846417746, "1": 0.2230209651},
+            "KendallRankCorrelation": {"0": -4.6080688829, "1": -0.8738917288},
+            "LogLikelihoodRatio": {"0": 2.264674102, "1": 1.7462342396},
+            "TTest": {"0": -1.0210127581, "1": -0.3685341442},
+            "TTestPValue": {"0": 0.2466906059, "1": 0.3876079751},
+        }
+    )
+
+
+@pytest.fixture(scope="session")
+def adult_data_distribution_balance_measures():
+    return pd.DataFrame(
+        {
+            "FeatureName": {"0": "race", "1": "gender"},
+            "KLDivergence": {"0": 0.2791392127, "1": 0.0576400449},
+            "JensenShannonDist": {"0": 0.2741838602, "1": 0.1209204479},
+            "WassersteinDist": {"0": 0.3549600737, "1": 0.168112715},
+            "InfiniteNormDist": {"0": 0.3549600737, "1": 0.168112715},
+            "TotalVarianceDist": {"0": 0.3549600737, "1": 0.168112715},
+            "ChiSquarePValue": {"0": 0.0, "1": 0.0},
+            "ChiSquareStat": {"0": 13127.8433660934, "1": 2944.6623157248},
+            "CrossEntropy": {"0": 1.5057745222, "1": 1.0865356576},
+        }
+    )
+
+
+@pytest.fixture(scope="session")
+def adult_data_aggregate_balance_measures():
+    return pd.DataFrame(
+        {
+            "TheilLIndex": {"0": 0.3912215368},
+            "TheilTIndex": {"0": 0.3420484048},
+            "AtkinsonIndex": {"0": 0.3237696704},
+        }
+    )
+
+
+@pytest.fixture(scope="session")
+def synthetic_data(
+    synthetic_data_label, synthetic_data_feature_1, synthetic_data_feature_2
+):
+    # https://github.com/microsoft/SynapseML/blob/4975dda59c8957270c64aa1eaf295673e14ac815/core/src/test/scala/com/microsoft/azure/synapse/ml/exploratory/DataBalanceTestBase.scala#L31
+    rows = [
+        (0, "Male", "Asian"),
+        (0, "Male", "White"),
+        (1, "Male", "Other"),
+        (1, "Male", "Black"),
+        (0, "Female", "White"),
+        (0, "Female", "Black"),
+        (1, "Female", "Black"),
+        (0, "Other", "Asian"),
+        (0, "Other", "White"),
+    ]
+    return pd.DataFrame(
+        rows,
+        columns=[
+            synthetic_data_label,
+            synthetic_data_feature_1,
+            synthetic_data_feature_2,
+        ],
+    )
+
+
+@pytest.fixture(scope="session")
+def synthetic_data_label():
+    return "Label"
+
+
+@pytest.fixture(scope="session")
+def synthetic_data_feature_1():
+    return "Gender"
+
+
+@pytest.fixture(scope="session")
+def synthetic_data_feature_2():
+    return "Ethnicity"
+
+
+@pytest.fixture(scope="session")
+def expected_aggregate_measures_feature_1():
+    # https://github.com/microsoft/SynapseML/blob/4975dda59c8957270c64aa1eaf295673e14ac815/core/src/test/scala/com/microsoft/azure/synapse/ml/exploratory/AggregateBalanceMeasureSuite.scala#L54
```

Comment:
I removed all the links in favor of this small comment block, which should clarify why they're there without any links

```
# This synthetic dataset was manually created in the SynapseML data balance
# feature so that metrics can be manually computed, thought of as the expected
# results, and then compared to the actual results. This and the expected
# measure fixtures below it are ports from the open source SynapseML repo.
```